app/views/cases/overturned_shared/_new_form.html.slim

```diff
@@ -7,36 +7,36 @@ section
   .heading-medium
     = "#{@case.subject} "
     span.no-break
-      = link_to @case.original_case_description,
+      = link_to @case.original_case_description + ' (opens in a new tab)',
               case_path(@case.original_case_id)
 
 hr
 
-.grid-row
-  .column-two-thirds
-    .form-group
-      = f.gov_uk_date_field :received_date, { legend_text: "Date Information Commissioner's Office (ICO) decision was received",
-              form_hint_text: '',
-              today_button: {class: 'button-secondary'} }
-
+fieldset#received_date
+  = f.hidden_field :received_date_dd
+  = f.hidden_field :received_date_mm
+  = f.hidden_field :received_date_yyyy
 
 .form-group
-  = f.gov_uk_date_field :external_deadline, { legend_text: "Final deadline" ,
-          form_hint_text: "We'll use this date to set the draft deadline, calculated as 20 working days from the final deadline."}
+  = f.gov_uk_date_field :external_deadline,
+                        { legend_text: "Final deadline" ,
+                          form_hint_text: "For example, 01 02 2018" }
+  .form-hint
```

Comment:
could you add a `p` in front of the class name.

Reply:
Yup!